paddle/phi/core/flags.h

```diff
@@ -24,34 +24,141 @@
 
 #include "paddle/utils/variant.h"
 
+#if defined(_WIN32)
+#if defined(BUILD_PHI_SHARED)
+#define PHI_EXPORT_FLAG __declspec(dllexport)
+#define PHI_IMPORT_FLAG __declspec(dllimport)
```

Comment:
`__declspec(dllexport)`
So is this the magic why we can export gflags symbols on the Windows platform?

Reply:
yes